WordPressKit/JetpackCapabilitiesServiceRemote.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// A service that returns the Jetpack Capabilities for a set of blogs
+open class JetpackCapabilitiesServiceRemote: ServiceRemoteWordPressComREST {
+
+
+    /// Returns a Dictionary of capabilities for each given siteID
+    /// - Parameters:
+    ///   - siteIds: an array of Int representing siteIDs
+    ///   - success: a success block that accepts a dictionary as a parameter
+    ///   - failure: a failure block
+    open func `for`(siteIds: [Int], success: @escaping ([String: AnyObject]) -> Void, failure: @escaping () -> Void) {
+
+        let endpoint = "rewind/batch-capabilities"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+
+        wordPressComRestApi.POST(path,
+                                 parameters: ["sites": siteIds] as [String: AnyObject],
+                                 success: {
+                                     response, _  in
```

Comment:
[nits] args should be on the previous line
```suggestion
                                 success: { response, _  in
```